bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -107,9 +111,7 @@ <h1>{{ self.page_title() }}</h1>
       )}}
 
       <div class="mzp-l-card-half">
-        <!-- 6 -->
```

Comment:
Is this removal intentional? Or a rebase gone wrong?

Reply:
@stephaniehobson 
It was intentional, I just felt like these comments aren't needed anymore seeing as the page is a lot less verbose and it is now easy to see which card number you are on with the new system.
(ex: `content_card('card_1')`, `content_card('card_2')`, .etc.)

Reply:
👍